pkgs/by-name/ys/ysfx/package.nix

```
@@ -0,0 +1,63 @@
+{ lib, stdenv, fetchFromGitHub, cmake, pkg-config
+, freetype, juce, libX11, libXcursor, libXext, libXinerama, libXrandr, libglvnd
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ysfx";
+  version = "2022-07-31";
```

Please read https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming
```suggestion
  version = "0-unstable-2022-07-31";
```

---

Done, thank you!